extensions/schema-compare/src/utils.ts

```diff
@@ -31,4 +32,20 @@ export function getTelemetryErrorType(msg: string): string {
 	else {
 		return 'Other';
 	}
+}
+
+/**
+ * Return the appropriate endpoint name depending on if the endpoint is a dacpac or a database
+ * @param endpoint endpoint to get the name of
+ */
+export function getEndpointName(endpoint: azdata.SchemaCompareEndpointInfo): string {
+	if (!endpoint) {
+		return undefined;
+	}
+
+	if (endpoint.endpointType === azdata.SchemaCompareEndpointType.Database) {
+		return `${endpoint.serverName}.${endpoint.databaseName}`;
```

Comment:
this works in most cases when we compare the previous and updated - but what if the connection difference was in user info?

Reply:
Any ideas on this? can we add other fields to check for?

Reply:
Maybe we need to also store the user. Looks like the server dropdown doesn't check user so the correct connection isn't always chosen so I'll fix that too.

Reply:
ok added check to see if user changed